Bychynski_Pavel_HW_8_2.py

```diff
@@ -0,0 +1,61 @@
+# 8.2 "Timer"
+class Timer:
+    def __init__(self, h = 0, m = 0, s = 0):
+        self.h = h
+        self.m = m
+        self.s = s
+        try:
+            assert self.h in range(0, 24) and self.m in range(0, 60) and self.s in range(0, 60)
+        except AssertionError:
+            print("incorrect values - setting default values")
+        except:
+            print("something where wrong - setting default values")
+
+    # def __str__(self):
+    #     return self.h, self.m,  self.s
+
+    def next_s(self):
+        if self.s != 59:
+            self.s += 1
+        else:
+            self.s = 0
+            if self.m != 59:
+                self.m += 1
+            else:
+                self.m = 0
+                if self.h != 23:
+                    self.h += 1
+                else:
+                    self.h = 0
```

Comment:
This method and others can be rewritten by the following algorithm:
```suggestion
def next_second(self):
        self.__seconds += 1
        if self.__seconds > 59:
            self.__seconds = 0
            self.__minutes += 1
            if self.__minutes > 59:
                self.__minutes = 0
                self.__hours += 1
                if self.__hours > 23:
                    self.__hours = 0
```